spec/models/response_template_spec.rb

```diff
@@ -27,4 +27,22 @@
       end
     end
   end
+
+  describe "user validation" do
+    it "validates the number of templates for a normal user" do
+      user = create(:user)
+      create_list(:response_template, 29, user_id: user.id)
```

Comment:
Shouldn't this be 30 because of "limit of 30"? and it should raise error on the 31st attempt?
```suggestion
      create_list(:response_template, 30, user_id: user.id)
```

Reply:
Good catch, had to update the validation as well